### ISSUE TITLE
Alternative hide/show shortcut key

### ISSUE THREAD

**Issue by sirmspencer** [flexsurfer/re-frisk#75]
For example use ctrl+g.

ctrl+h is a common key for plug-ins.  Allowing it to change would help with conflicts.

**Comment by flexsurfer** [comment]
i believe `ctrl+h ` is used in re-frame-10x, do you know how it works for your team who works with it ?

**Comment by sirmspencer** [comment]
Yes, ctrl+h is used in re-frame-10x.  It means we cant load both plugins at the same time.

**Comment by sirmspencer** [comment]
Apparently ctrl+h is also the history shortcut for firefox, so that interferes as well.

**Comment by sirmspencer** [comment]
Updated title.  I was thinking that it would be configurable and not just changed to a different key that could be a different conflict.

**Comment by flexsurfer** [comment]
> Apparently ctrl+h is also the history shortcut for firefox, so that interferes as well.

but how it works for re-frame-10x then? do they have a solution ?

**Comment by sirmspencer** [comment]
> > Apparently ctrl+h is also the history shortcut for firefox, so that interferes as well.
> 
> but how it works for re-frame-10x then? do they have a solution ?

Good question, they cannot use firefox, so they use chrome.

**Comment by flexsurfer** [comment]
yes sure, PRs are welcomed , thank you
